Makefile

```diff
@@ -9,8 +9,8 @@ cert_dir=$(HOME)/.nextcloud/certificates
 github_account=nextcloud
 release_account=nextcloud-releases
 branch=master
```

Comment:
```suggestion
branch=stable26
```
🤔

Reply:
not that important in fact, since release is scripted :)